Dockerfile

```diff
@@ -5,7 +5,7 @@ COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -o /out/linkerd2-proxy-init -mod=readonly -ldflags "-s -w" -v
 
 ## package runtime
-FROM debian:stretch-20190812-slim
+FROM debian:buster-20190910-slim
```

Comment:
Should this perhaps be `buster-**20200514**-slim` in order to match the buster version used in the [linkerd2](https://github.com/linkerd/linkerd2) repo (https://github.com/linkerd/linkerd2/pull/4542/files)?